docs/cloud/pouta/tutorials/image-pipeline.md

```diff
@@ -0,0 +1,650 @@
+| operating_status         |                                      |
+| public                   | True                                 |
+| region                   | regionOne                            |
+| service_status_updated   | 2025-02-04T13:08:51                  |
+| status                   | BUILD                                |
+| updated                  | 2025-02-04T13:08:51                  |
+| volume                   | 1                                    |
++--------------------------+--------------------------------------+
+```
+
+In the rest of the tutorial, we assume that:
+
+* the name of the database instance in Pukki is `pipeline_db_instance`
+* the name of the postgresql database is `pipeline_db`
+* the username for the admin of the postgresql database is `db_admin`
+* the password for the admin of the postgresql database is `db_password`
+
+## Step 3: creating virtual machine in cPouta
+
+We open a third terminal window, to which we will refer using the name `terminal_C`.
+We use `terminal_C` for all the commands dealing with cPouta.
+
+In the same way as for Allas and Pukki, to continue the tutorial we need to have a working command-line interface for cPouta as well.
+We follow the [instructions on how to install and configure cPouta command-line interface](../install-client.md), if we haven't done it already.
+
+We can test the correct functioning of the command-line interface by, for example, showing the properties of one of the flavors.
+An example of the command and its expected output follows:
+```
+$ openstack flavor show standard.tiny
++----------------------------+--------------------------------------+
+| Field                      | Value                                |
++----------------------------+--------------------------------------+
+| OS-FLV-DISABLED:disabled   | False                                |
+| OS-FLV-EXT-DATA:ephemeral  | 0                                    |
+| access_project_ids         | None                                 |
+| disk                       | 80                                   |
+| id                         | 0143b0d1-4788-4d1f-aa04-4473e4a7c2a6 |
+| name                       | standard.tiny                        |
+| os-flavor-access:is_public | True                                 |
+| properties                 | standard='true'                      |
+| ram                        | 1000                                 |
+| rxtx_factor                | 1.0                                  |
+| swap                       |                                      |
+| vcpus                      | 1                                    |
++----------------------------+--------------------------------------+
+```
+
+First, we create a keypair, which we will use to access the virtual machine once it is up and running.
+To create a new keypair, we run the following command:
+```
+openstack keypair create mykeypair > mykeypair.pem
+```
+
```

Comment:
You can add a info box to link to the web interface method of creating a key pair.

Reply:
To the documentation we have that explains it.

Reply:
After discussion, decided to drop adding links to gui alternatives